File: Python/Math-Array-String/41.py
# 2769. Find the Maximum Achievable Number


class Solution:
    def theMaximumAchievableX(self, num: int, t: int) -> int:
        return num + 2 * t


sol = Solution()
output = sol.theMaximumAchievableX(4, 1)
print(output)

File: JavaScript/Math-Array-String/41.js
// 2769. Find the Maximum Achievable Number


/**
 * @param {number} num
 * @param {number} t
 * @return {number}
 */
var theMaximumAchievableX = function(num, t) {
    return num + 2 * t;
};

// Examples
console.log(theMaximumAchievableX(4, 1)); // Output: 6
console.log(theMaximumAchievableX(3, 2)); // Output: 7
